apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -86,24 +84,22 @@ function TokenDetails({ coinType }: TokenDetailsProps) {
                 className="flex flex-col h-full flex-1 flex-grow items-center"
                 data-testid="coin-page"
             >
-                {isError ? (
+                {!coinType && <AccountSelector />}
+                <div className="mt-1.5">
+                    <CoinBalance
+                        balance={tokenBalance}
+                        type={activeCoinType}
+                        mode="standalone"
+                    />
+                </div>
+                {isError || errorUpdateCount > 0 ? (
```

Comment:
Why use `errorUpdateCount` and why not just `isError`?

Reply:
just prevent error message from diaappearing while the refetching. isError/error changes state during this time. I will remove it